public/controllers/agent/register-agent/steps/__snapshots__/server-address.test.tsx.snap

```diff
@@ -0,0 +1,84 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Server Address Combobox should match snapshot 1`] = `
+<div>
+  <div
+    class="euiText euiText--medium"
+  >
+    <p>
+      This is the address the agent uses to communicate with the Wazuh server. It can be a list or an IP address or a fully qualified domain name (FQDN).
```

Comment:
This is the address the agent uses to communicate with the Wazuh server. It can be a list of one or more IP addresses or fully qualified domain names (FQDN).